app/models/manager_refresh/inventory/middleware_manager.rb

```diff
@@ -0,0 +1,62 @@
+module ManagerRefresh::Inventory::MiddlewareManager
+  extend ActiveSupport::Concern
+  include Core
+
+  COMMON_ATTRIBUTES = %i(name ems_ref nativeid properties feed).freeze
+
+  class_methods do
+    def has_middleware_manager_domains(options = {})
```

Comment:
I'm not sure, personally I'd choose something else, but for consistency sake I think we should just disable the rubocop alerts.